tritium-lib/src/test/java/com/palantir/tritium/proxy/InvocationEventProxyTest.java

```diff
@@ -132,12 +132,12 @@ public void onSuccess(@Nullable InvocationContext context, @Nullable Object resu
     public void testInstrumentOnFailureThrows() throws Throwable {
         InvocationEventHandler<InvocationContext> testHandler = new SimpleHandler() {
             @Override
-            public void onSuccess(@Nullable InvocationContext context, @Nullable Object result) {
+            public void onSuccess(@Nullable InvocationContext unusedContext, @Nullable Object unusedResult) {
```

Comment:
Does SuppressWarnings(“unused”) work? It feels odd to change parameter names from the interface.

Reply:
yeah, `@SuppressWarnings("unused")` or `@SuppressWarnings("StrictUnusedVariable")` will suppress the warning (error), though in general I would want to avoid that at the method level to avoid accidentally suppressing legit problems.

Reply:
i _think_ you can also put that annotation on a parameter, not a method.

Reply:
yep, `@SuppressWarnings("unused")` on the param itself also works. Not sure how I feel about that everywhere, it will definitely re-wrap a bunch of args, especially for what are often no-op callbacks, for example:


```
        @Override
        public void onSuccess(
                @SuppressWarnings("unused") @Nullable InvocationContext context,
                @SuppressWarnings("unused") @Nullable Object result) {}
```

vs.

```
        @Override
        public void onSuccess(@Nullable InvocationContext unusedContext, @Nullable Object unusedResult) {}
```